src/community/geopkg/src/main/java/org/geoserver/geopkg/GeoPackageGetMapOutputFormatWPS.java

```diff
@@ -1,190 +1,34 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
- * (c) 2001 - 2013 OpenPlans
- * This code is licensed under the GPL 2.0 license, available at the root
- * application directory.
- */
 package org.geoserver.geopkg;
 
-import static org.geoserver.geopkg.GeoPkg.EXTENSION;
-import static org.geoserver.geopkg.GeoPkg.MIME_TYPE;
-import static org.geoserver.geopkg.GeoPkg.NAMES;
+import static org.geoserver.geopackage.GeoPkg.*;
 
-import com.google.common.collect.Sets;
-import java.io.File;
 import java.io.IOException;
 import java.util.List;
 import java.util.Map;
 import java.util.SortedMap;
 import java.util.TreeMap;
 import java.util.logging.Level;
-import org.geoserver.catalog.ResourceInfo;
 import org.geoserver.gwc.GWC;
 import org.geoserver.ows.util.OwsUtils;
 import org.geoserver.platform.GeoServerExtensions;
 import org.geoserver.platform.ServiceException;
-import org.geoserver.tiles.AbstractTilesGetMapOutputFormat;
-import org.geoserver.wms.GetMapRequest;
-import org.geoserver.wms.MapLayerInfo;
-import org.geoserver.wms.RasterCleaner;
-import org.geoserver.wms.WMS;
-import org.geoserver.wms.WMSMapContent;
-import org.geoserver.wms.WebMap;
-import org.geoserver.wms.WebMapService;
+import org.geoserver.wms.*;
```

Comment:
I thought a PMD check would catch this, but perhaps not run for community modules?

Reply:
QA does not run in community modules, indeed. On move to extension you'll face the full force of the QA checks :-D